indico/modules/events/timetable/templates/display/_common.html

```diff
@@ -0,0 +1,62 @@
+{% from 'events/notes/_note.html' import render_note %}
+{% from 'attachments/_display.html' import render_attachments_folders %}
+{% from 'events/display/_common.html' import render_users %}
+
+{% macro render_description(item) %}
+    <span itemprop="description" class="contrib-description html">
```

Comment:
the `html` class can go away, it was a no-op and just for consistency with the `text` class (for preformatted display)